qa/L0_model_config/autofill_noplatform/python/conflicting_max_batch_size/expected

```diff
@@ -0,0 +1 @@
+configuration specified max_batch_size 2 but Python model specified max_batch_size 4
```

Comment:
Referring to the comment here: https://github.com/triton-inference-server/python_backend/pull/138#discussion_r870840585 I was under the impression that if max_batch_size has a confliction, we should throw an error. Should I change the logic so that it won't throw an error when the max_batch_size specified in config.pbtxt is less than the max_batch_size specified in model.py?

Reply:
Yes. I think the user might want to tune this value separately. And we must not throw error if batch_size is within max_batch_size provided by the model.

Reply:
I see. Will modify the logic, thanks.